README.md

```diff
@@ -23,6 +23,10 @@ See [INSTALL.md](https://github.com/facebook/pyre-check/blob/master/INSTALL.md)
 
 See [CONTRIBUTING.md](https://github.com/facebook/pyre-check/blob/master/CONTRIBUTING.md) for how to help out.
 
+## Languages
```

Comment:
One small detail: Could you change the format of linking to mimic the structure in https://github.com/tiimgreen/github-cheat-sheet/blob/master/README.md, and have [Español] as the link name?

Reply:
Thank you  :)

Reply:
I will update it.